src/main.rs

```diff
@@ -112,7 +112,26 @@ async fn start(
 		session_tx,
 	)
 	.wrap_err("failed to start compositor")?;
+
 	sleep(Duration::from_millis(2000)).await;
```

Comment:
Note: I've not committed it, but I experimentally remove this `sleep()` and don't seem to have any issues, so I wonder if this was a workaround for a sequence issue that is now resolved by calling `systemctl` later?

Reply:
Perhaps @wash2 remembers? ( https://github.com/pop-os/cosmic-session/commit/23d702ae1e8325ce8a7ed3851d1a280019a268d8 )

Reply:
It probably was to make sure the WAYLAND_DISPLAY variable was set before virtwl-proxy was started back in the day.

Feel free to drop it, if it doesn't cause any issues.